python_ta/checkers/pycodestyle_checker.py

```diff
@@ -35,16 +35,16 @@ def process_module(self, node: nodes.NodeNG) -> None:
         )
         report = style_guide.check_files()
 
-        for line_num, msg in report.get_file_results():
-            self.add_message("pep8-errors", line=line_num, args=msg)
+        for line_num, msg, code in report.get_file_results():
+            self.add_message("pep8-errors", line=line_num, args=(code, msg))
 
 
 class JSONReport(pycodestyle.StandardReport):
     def get_file_results(self) -> List[Tuple]:
         self._deferred_print.sort()
         return [
-            (line_number, f"line {line_number}, column {offset}: {text}")
-            for line_number, offset, _, text, _ in self._deferred_print
+            (line_number, f"line {line_number}, column {offset}: {text}", f"{code}")
```

Comment:
The f-string for `code` is unnecessary, as I imagine `code` is already a string

Reply:
Thank you! I have fixed this just now.